source/system/system.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package system
+
+import (
+	"fmt"
+	"sigs.k8s.io/node-feature-discovery/source"
+)
+
+// Source implements FeatureSource.
+type Source struct{}
+
+// Name returns an identifier string for this feature source.
+func (s Source) Name() string { return "system" }
+
+// Discover returns feature names for system configuration: irqbalance, swap, ksm, memory compaction, iommu...
+func (s Source) Discover() (source.Features, error) {
+	features := source.Features{}
+	active, err := SystemctlStatusIsActive("irqbalance")
```

Comment:
As I commented before, I think the list of services to detect should be configurable, allowing the detection of other system services, too. irqbalance can be the default. You can take a look at kernel config detection to see an example.

Reply:
Hmmm...

Reply:
Can we just use this method:
for _, attrib := range []string{"irqbalance", "openswitch"} {
...
Just like the same method in #195  :)